en/docs/get-started/quick-start-guide.md

```diff
@@ -104,19 +103,50 @@ Follow this procedure to further develop the API by adding a new resource and si
 
      ![Configure Variable](../assets/img/get-started/configure-variable.png){.cInlineImage-full}
 
-8. Click **Save**.
+ 7. Click **Save**.
 
-9. To return the personalized message, click **+** below the variable you created, click **Return**, and enter `message` as the **Return Expression**.
-   ![Add Return Type](../assets/img/get-started/add-return-type.png){.cInlineImage-full}
-   ![Select Return](../assets/img/get-started/select-return.png){.cInlineImage-full}
+ 8. To return the personalized message, click **+** below the variable you created, click **Return**, and enter `message` as the **Return Expression**.
+    ![Add Return Type](../assets/img/get-started/add-return-type.png){.cInlineImage-full}
+    ![Select Return](../assets/img/get-started/select-return.png){.cInlineImage-full}
    
-10. Click **Save**.
+ 9. Click **Save**.
 
 !!!info
-    You can click the **</>** icon on the top left corner of the editor if you want to work in the code view. Depending on your expertise, you can choose to work in the low-code view or pro-code view, or both simultaneously.    
+    You can click the **</>** icon on the top left corner of the editor if you want to work in the code view. Depending on your expertise, you can work in the low-code view or the pro-code view. You can also work in both views simultaneously.    
     ![Editor View](../assets/img/get-started/low-code-pro-code-side-by-side.png){.cInlineImage-full}
 
-### Step 2: Commit Changes to GitHub
+### Step 2: Run and test the REST API
+
+Once you update the REST API configuration as described above, you can verify whether it works as expected by following these steps:
+
+1. Click **Run**.
+
+    ![Run the REST API](../assets/img/get-started/run-the-rest-api.png){.cInlineImage-half}
+
+    A terminal opens with logs that indicate the status of the REST API. If you updated the REST API as instructed above, you will see the following log:
+
+    ```
+    Running executable
+    ```
+2. To invoke the REST API, click **Try it**.
+   
+    ![Try REST API](../assets/img/get-started/try-rest-api.png){.cInlineImage-half}
+
+    This opens the Swagger view with details of your REST API.
+
+3. Expand the GET resource and click **Try it out**.
+
+    ![Try out REST API](../assets/img/get-started/try-out-rest-api.png){.cInlineImage-full}
+
+4. In the **name** field, enter any value (e.g., `Alice`).
```

Comment:
The Google style guide recommends the use of `for example` instead of `e.g.,`. Shall we go with that approach?
https://developers.google.com/style/word-list?hl=en#ecommerce

Reply:
Done